Sloth.Core/Extensions/DateTimeExtensions.cs

```diff
@@ -22,54 +22,24 @@ public static DateTime FromPacificTime(this DateTime dateTime)
         }
 
         /// <summary>
-        /// Returns the Fiscal Year according to the date provided to be used for transactions.
+        /// Fiscal Year runs from July 1 to Jun 30
         /// </summary>
-        /// <returns>Fiscal Year</returns>
-        public static int FiscalYear(this DateTime datetime)
+        /// <param name="date"></param>
+        /// <returns></returns>
+        public static int GetFinancialYear(this DateTime date)
         {
-            if (datetime == null)
-            {
-                throw new ArgumentNullException(nameof(datetime));
-            }
-
-            // Fiscal dates for KFS are PST
-            datetime = datetime.ToPacificTime();
-
-            var year = datetime.Year;
-            var month = datetime.Month;
-
-            // Use the current year if prior to July 1st;
-            // Otherwise use the next calendar year.
-            if (month < 7)
-                return year;
-
-            return (year + 1);
+            return date.Month >= 7 ? date.Year + 1 : date.Year;
         }
 
         /// <summary>
-        /// Returns the Fiscal Period according to date provided to be used for transactions.
+        ///  Fiscal Period is the month designator for the fiscal year.
+        ///    So Jul = 1, Dec = 6, Jan = 7, Jun = 12
         /// </summary>
-        /// <returns>Fiscal Period string</returns>
-        public static int FiscalPeriod(this DateTime datetime)
+        /// <param name="date"></param>
+        /// <returns></returns>
+        public static int GetFiscalPeriod(this DateTime date)
         {
-            if (datetime == null)
-            {
-                throw new ArgumentNullException(nameof(datetime));
-            }
-
-            // Fiscal dates for KFS are PST
-            datetime = datetime.ToPacificTime();
```

Comment:
@sprucely Should we keep the PST?

Reply:
This extension method wasn't even being used. So I think not.